spark/v3.5/spark/src/test/java/org/apache/iceberg/spark/actions/TestCreateActions.java

```diff
@@ -94,7 +95,7 @@ public class TestCreateActions extends SparkCatalogTestBase {
 
   private static final String NAMESPACE = "default";
 
-  @Parameterized.Parameters(name = "Catalog Name {0} - Options {2}")
+  @Parameters(name = "Catalog Name {0} - Options {2}")
```

Comment:
isn't this missing the `type` parameter here with index 3?

Reply:
yeah, will update this and review all other files Parameters name